mainlyetcetera/rotten_tomatillos
# Refactor/rename container

## Is this a fix or a feature?
- refactor
## What is the change/fix?
- change `Container` component to `Display`
## Where should the reviewer start?
- `src/components/Display/Display.js`
## How should this be tested?
- open app, make sure it all still works
- run `npm run cypress`
- ensure all tests still passing
## This PR is related to which issues:
Closes #31
